gradle.properties

```diff
@@ -34,7 +34,7 @@ mod.name=Flux Networks
 # The license of the mod. Review your options at https://choosealicense.com/
 mod.license=MIT
 # The mod version. See https://semver.org/
-mod.version=7.2.2
+mod.version=8.3.3
```

Comment:
Please don't bump the version like that. Semver exists for a reason.

Reply:
Ok

Reply:
Would you mind giving permission to upload